### DISCUSSION TITLE
Webhook is not being triggered anymore

### DISCUSSION THREAD

**Discussion by muezz** [supabase/supabase#14028] [Questions]
In my project, I have a webhook that gets triggered on update of a table and it then triggers another Edge Function. This flow was working perfectly a week ago and I noticed a few days back that it is no longer working. I am not sure how to debug this exactly. Can anyone tell me what I should do?

**Comment by silentworks** [comment]
Have you tried hitting the Edge Function directly with something like curl, Postman or Insomnia? Also you can check the [Postgres logs](https://app.supabase.com/project/mkznoxjiwtfuegwckmnq/logs/postgres-logs) and the [Edge Function itself logs](https://app.supabase.com/project/mkznoxjiwtfuegwckmnq/functions).

**Reply by muezz** [reply]
Yes. The edge function works on its own as well as from the webhook. I have checked the logs and haven't found anything so far.

The day before yesterday, I tried it again and it was working then. I did not do anything apart from creating a new webhook (with the same config to see if that worked). I deleted the new webhook and the old one was also working. I have no idea why it stopped working for about 4-5 days. I am 100% sure that it stopped working and I am also 100% sure that I had not made any changes from when it was working earlier and when it started working again.

**Reply by silentworks** [reply]
If you should find this happening again please contact support and they can take a look into what may be causing that. https://app.supabase.io/support/new

**Reply by muezz** [reply]
I did contact them and I got a response that asked me to try a few basic things like running the edge function directly and checking if there were any changes since the last time it was working.

I replied with the relevant information and then I never heard from them.
